platform/android/display_server_android.h

```diff
@@ -123,6 +123,7 @@ class DisplayServerAndroid : public DisplayServer {
 
 	virtual void screen_set_orientation(ScreenOrientation p_orientation, int p_screen = SCREEN_OF_MAIN_WINDOW) override;
 	virtual ScreenOrientation screen_get_orientation(int p_screen = SCREEN_OF_MAIN_WINDOW) const override;
+	virtual int screen_get_current_rotation(int p_screen) const override;
```

Comment:
This should be implemented for the rest of platforms (should be easy to do an all except Web).